lib/capnp_c.h

```diff
@@ -22,6 +28,19 @@
 typedef intmax_t ssize_t;
 #endif
 
+// Macro UNUSED(x) silences compiler warnings about unused parameter or variable `x`.
+#ifndef UNUSED
+#define UNUSED(x) (void)(x)
+#endif
+
+// Cross-platform macro ALIGNED_(x) aligns a struct by `x` bits.
```

Comment:
Yes, the attribute should stay - but the value needs to be 8 for GCC. I'm not sure if MSVC expects bits or bytes, it's possible one of the macros needs a `* 8` or `/ 8`...